build/charts/antrea/conf/antrea-agent.conf

```diff
@@ -346,6 +346,12 @@ multicluster:
   enableStretchedNetworkPolicy: {{ .enableStretchedNetworkPolicy }}
 # Enable Pod to Pod connectivity.
   enablePodToPodConnectivity: {{ .enablePodToPodConnectivity }}
+# WireGuard tunnel configuration for cross-cluster traffic.
+  wireGuard:
+    # Enable WireGuard tunnel for cross-cluster traffic.
+    enable: {{ .wireGuard.enable }}
```

Comment:
Question: Is it possible that the cross-cluster traffic could be encrypted in another way in the future?
If yes, I feel this should use a configuration similar to `trafficEncryptionMode` to be extensible, like `multicluster.trafficEncryptionMode`.

Reply:
I remember Lan @luolanzone  had mentioned this before, maybe `trafficEncryptionMode` is better. Thanks for the advice.

Reply:
Done. Use `trafficEncryptionMode` now.